projects/angular/src/wizard/wizard.ts

```diff
@@ -47,6 +48,11 @@ import { ClrWizardTitle } from './wizard-title';
   },
 })
 export class ClrWizard implements OnDestroy, AfterContentInit, DoCheck {
+  /**
+   * Set the aria-label for the stepnav section of the wizard. Set using `[clrWizardStepnavAriaLabel]` input.
+   */
+  @Input('clrWizardStepnavAriaLabel') stepnavAriaLabel = commonStringsDefault.wizardStepnavAriaLabel;
```

Comment:
We need to inject the common strings service and the get the value from there so that it is translated if needed.